snapshot.go

```diff
@@ -9,12 +9,12 @@ package knoxite
 
 import (
 	"encoding/json"
+	uuid "github.com/nu7hatch/gouuid"
```

Comment:
I prefer having a separate block of external imports below, as it used to be. No dealbreaker though.